### PR TITLE
role salary slider

### PR DESCRIPTION
**Pull request by JonathanWren**

### CODE REVIEW

File: main/rolesalary.js [hunk 1]
```diff
@@ -9,42 +9,78 @@ import { useDispatch, useSelector } from 'react-redux';
...
+          slider += salaryRemaining/salaryArray[section][1];
+          salaryRemaining=0;
+        }
+      }
+      return slider;
     }
 
-export default RoleSalary;
\ No newline at end of file
+    const [sliderValue, setSliderValue] = useState(sliderCalculator(initialSalary));
```
Comment (JonathanWren) [inline main/rolesalary.js:37]: I do not think that this is necessary - surely  can have an initialSliderValue that is set when the component is first loaded.

File: main/rolesalary.js [hunk 2]
```diff
@@ -9,42 +9,78 @@ import { useDispatch, useSelector } from 'react-redux';
...
 
 
-const RoleSalary = () => {
-    
-        const nav = useNavigation();
-        const dispatch = useDispatch();
-    
-        const initialSalary = useSelector(state => state.profile.nextMove.salary);
-        const [salary, setSalary] = useState(initialSalary);      
+const SalarySlider = () => {
```
[the remark this answers is not in the document]
Comment (Caleb-Agnes) [reply main/rolesalary.js:12]: Yes that makes a lot of sense, it is my fault from copying and pasting it across from the test version without changing the name back

File: main/rolesalary.js (hunk 1 above)
Comment (Caleb-Agnes) [reply main/rolesalary.js:37]: in theory that sounds good but I might need some help implementing that as I'm not sure how that would work.

Comment (JonathanWren) [reply main/rolesalary.js]: @Caleb-Agnes I have made this change. If you pull from the remote repository you should be able to see the changes.

**Review by Caleb-Agnes — APPROVED** [review]

**Pull request merged**